assets/less/content/general.less

```diff
@@ -32,6 +32,10 @@ h2 {
   font-size: 1.625em;
   margin: 1em 0 .5em;
   font-weight: 400;
+
+  &.section-heading {
+    margin: 1.5em 0 .5em .25em;
```

Comment:
This fixes a small mis-alignment but not sure if there is a better way to do this.